modules/configuring-access-control-and-sending-invitations.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+//
+// * configuration/inviting-users-to-rhacs.adoc
+
+:_content-type: PROCEDURE
+[id="configuring-access-control-and-sending-invitations_{context}"]
+= Configuring access control and sending invitations
+
+.Procedure
+
+. To invite a user, use one of the following methods:
+* In the {product-title-short} portal, navigate to *Platform Configuration -> Access Control -> Auth providers* tab, and then click *Invite users*.
+* Navigate to your profile icon, and then click *Invite users*.
+
+. In the *Invite users* dialog box, provide the following information:
+* *Emails to invite*: Enter one or more email addresses of the users you want to invite. Ensure that they are valid email addresses associated with the intended recipients.
+* *Provider*: From the drop-down list, select a provider you want to use for each invited user. For {product-title-managed-short}, the default option is `Default Internal SSO`.
```

Comment:
This is not quite accurate.

- If there is only one auth provider available, it will be pre-selected.
- If there is more than one auth provider available, and one of them is "Red Hat SSO" or "Default Internal SSO", that one will be preselected.
- If there is more than one auth provider available, and none are "Red Hat SSO" or "Default Internal SSO", then the user will have to select one.

Note: if no auth providers have been set up yet, then the user is shown a warning, the form is disabled, and there is a link to take the user over to the Access Control section where they can set up an auth provider.